lib/routes/devices-sessions.js

```diff
@@ -317,6 +317,7 @@ module.exports = (log, db, config, customs, push, devices) => {
             id: isA.string().regex(HEX_STRING).required(),
             lastAccessTime: isA.number().min(0).required().allow(null),
             lastAccessTimeFormatted: isA.string().optional().allow(''),
+            location: isA.object().optional().allow(null),
```

Comment:
Looks like `allow(null)` is unnecessary here, the code below ensures there is always a location object?

Also can we validate the keys for `location`, `state` and `country`.